include/openthread/nat64.h

```diff
@@ -226,6 +226,65 @@ otError otNat64GetNextAddressMapping(otInstance *                   aInstance,
                                      otNat64AddressMappingIterator *aIterator,
                                      otNat64AddressMapping *        aMapping);
 
+/**
+ * States of NAT64.
+ *
+ */
+typedef enum
+{
+    OT_NAT64_STATE_DISABLED = 0, ///< NAT64 is disabled.
+    OT_NAT64_STATE_NOT_RUNNING,  ///< NAT64 is enabled, but is not in function.
+    OT_NAT64_STATE_IDLE,   ///< NAT64 is enabled, but the border router is not an active NAT64 border router in the
```

Comment:
Can we make the comment shorter to fit in same line?
Maybe something like  `///< NAT64 enabled but there is no active NAT64 BR.`